src/private/api/api.js

```diff
@@ -0,0 +1,71 @@
+const ERROR = require('../errors.js');
+const {
+  verifyToken,
+  logUserActivity
+} = require('../middleware.js');
+const {
+  respondToClient
+} = require('../response.js');
```

Comment:
Also, given that you've now pulled master onto this branch, might be worth running prettify with `npm run prettify`.